packages/expect/src/types.ts

```diff
@@ -58,7 +57,7 @@ export interface MatcherUtils {
 
 export interface MatcherState {
   assertionCalls: number;
-  currentConcurrentTestName?: AsyncLocalStorage<string>;
+  currentConcurrentTestName?: () => string | undefined;
```

Comment:
Looks like breaking change, or?

Reply:
yes. Not sure if it's breaking _enough_, tho... 😅

Reply:
Well, breaking for anyone that calls `jestExpect.setState` with this property or anyone that reads it inside a custom matcher. Since the property was only added in v29.6.0 then I'm pretty sure only Jest itself does that 😅